assets/javascripts/modules/checkout/eventTracking.js

```diff
@@ -0,0 +1,36 @@
+define(['modules/analytics/ga', 'modules/checkout/ratePlanChoice'], function (ga, ratePlanChoice) {
+    'use strict';
+
+    var STARTED = Date.now();
+
+    function getElapsedTimeMs() {
```

Comment:
Does this not return time in seconds not milliseconds? If so should the function name be altered to reflect this?